noise.hoc

```diff
@@ -5,6 +5,7 @@ proc init() {
 pp=new IClamp(0.5)
 pp.dur=5000
 r=new Random($1)
+r.Random123()
```

Comment:
I think all of these changes should be of the form
```r.Random123($1)```

Reply:
All in this particular file, or all in the entire PR?